src/Symfony/Component/Security/composer.json

```diff
@@ -48,6 +48,7 @@
         "symfony/expression-language": "For using the expression voter",
         "ircmaxell/password-compat": "For using the BCrypt password encoder in PHP <5.5"
     },
+    "minimum-stability": "dev",
```

Comment:
For the minimum-stability, I don't think so. It's simply a remnant of the previous PR, so I guess I'll fix that. No problem for the LDAP suggest (for the extension), too. :)

As for the SecurityExtension triggering an exception if the LDAP extension is not installed, I don't think it's necessary. If you wish to, you can use the PEAR LDAP2 implementation, which is a full-PHP implementation. And the default implementation (which uses the LDAP extension) already throws an exception if the extension is not loaded. :)